### PR TITLE
Turn off Display and visualization by default

### PR DESCRIPTION
**Pull request by cociweb**
Turn off Display support and use None type visualization by default. This PR tries to resolves the periodical "unhandled grfe" messages in the player's log as defined in music-assistant/hass-music-assistant/issues/2118

### CODE REVIEW

**Comment by cociweb** [comment]
I'm still getting:
```
[23:25:03.913442] process:537 unhandled grfb
[23:25:03.931492] process:537 unhandled grfe
```

but only 1 time: at the startup of the client. 
The periodical flood is resolved by this PR. - I think we can live with it. 😉

(Just for the record: [grfb](https://github.com/home-assistant-libs/aioslimproto/blob/49893022a5df01a23f1bbbfb8847f5fc1860759f/aioslimproto/display.py#L188C1-L195C42) is for the brightness, [grfe](https://github.com/home-assistant-libs/aioslimproto/blob/49893022a5df01a23f1bbbfb8847f5fc1860759f/aioslimproto/display.py#L164C1-L186C52) is for rendering)

File: music_assistant/server/providers/slimproto/__init__.py [hunk 1]
```diff
@@ -107,7 +107,7 @@ class SyncPlayPoint:
 
 DEFAULT_PLAYER_VOLUME = 20
 DEFAULT_SLIMPROTO_PORT = 3483
-DEFAULT_VISUALIZATION = SlimVisualisationType.SPECTRUM_ANALYZER.value
+DEFAULT_VISUALIZATION = SlimVisualisationType.NONE.value
```
Comment (marcelveldt) [inline music_assistant/server/providers/slimproto/__init__.py:110]:
```suggestion
DEFAULT_VISUALIZATION = SlimVisualisationType.NONE
```

File: music_assistant/server/providers/slimproto/__init__.py [hunk 2]
```diff
@@ -905,10 +905,10 @@ async def _set_preset_items(self, slimplayer: SlimClient) -> None:
     async def _set_display(self, slimplayer: SlimClient) -> None:
         """Set the display config for a player."""
         display_enabled = self.mass.config.get_raw_player_config_value(
-            slimplayer.player_id, CONF_DISPLAY, True
+            slimplayer.player_id, CONF_ENTRY_DISPLAY.key, CONF_ENTRY_DISPLAY.default_value
         )
         visualization = self.mass.config.get_raw_player_config_value(
-            slimplayer.player_id, CONF_VISUALIZATION, DEFAULT_VISUALIZATION
+            slimplayer.player_id, CONF_ENTRY_VISUALIZATION.key, CONF_ENTRY_VISUALIZATION.default_value
```
Comment (marcelveldt) [inline music_assistant/server/providers/slimproto/__init__.py:911]: please lint this line, its too long

**Comment by marcelveldt** [comment]
Check the lint issues - otherwise good!

**Review by marcelveldt — APPROVED** [review]
Thanks!

**Pull request merged**